website/docs/r/validating_webhook_configuration.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: validating_webhook_configuration"
+sidebar_current: "docs-kubernetes-resource-validating-webhook-configuration"
+description: |-
+  Validating Webhook Configuration automatically scales the number of pods in a replication controller, deployment or replica set based on observed CPU utilization.
```

Comment:
Is this the right description for the `validating_webhook_configuration` resource?